bpython/test/test_import_not_cyclical.py

```diff
@@ -0,0 +1,108 @@
+from bpython.test import unittest
+from bpython.importcompletion import find_modules
+import os, sys, tempfile
+
+
+class TestAvoidSymbolicLinks(unittest.TestCase):
+    def setUp(self):
+        with tempfile.TemporaryDirectory() as import_test_folder:
+            try:
+                os.chdir(import_test_folder)
+            except:
+                print("Can't change the Current Working Directory")
+                import_test_folder.cleanup()
+                sys.exit()
+
+            os.mkdir(os.path.join(os.getcwd(), "Level0"))
+            os.mkdir(os.path.join(os.getcwd(), "Right"))
+            os.mkdir(os.path.join(os.getcwd(), "Left"))
+            try:
+                os.chdir("Level0")
+                with open("__init__.py", "w") as init_file:
+                    init_file.close()
```

Comment:
A `pass` is sufficient here. The file will be closed when exiting this block in any case.